package.json

```diff
@@ -12,7 +12,8 @@
         "test:qunit": "grunt qunit",
         "test:playwright": "playwright test \"--config=tests/playwright/config/sanity.config.js\"",
         "test:playwright:debug": "npm run test:playwright -- --debug",
-        "dev:start": "npm install && composer install",
+		"test:playwright:local": "playwright test \"--config=tests/playwright/config/local.config.js\"",
```

Comment:
Maybe add a comment about this in the task?
I will add a comment about it in the commit message when this PR is squashed and merged.

Reply:
Done